waggle-dance-extensions/src/main/java/com/hotels/bdp/waggledance/extensions/client/ratelimit/RateLimitingInvocationHandler.java

```diff
@@ -71,6 +76,7 @@ private Object doRateLimitCall(CloseableThriftHiveMetastoreIface client, Method
     if (shouldProceedWithCall(method)) {
       return doRealCall(client, method, args);
```

Comment:
If you add a metric here named `X`, you can replace the `CALLS` metric with the sum of `EXCEEDED` + `X`. (Sorry, I can't think of a good name).